index.html

```diff
@@ -431,7 +440,7 @@
 
       <p>Pointer lock must be exited if the
       <a>target</a> is removed from its document, or the <a>user agent</a>,
-      window, or tab loses focus. Moving focus between elements of
+      window, or tab loses focus. Moving focus between elements whose shadow-including root is
```

Comment:
Could you re-check this sentence? I does not look correct to me.

s/active documents/active document/ ???